dev-libs/libucl/libucl-0.7.3.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Do you need to revive this old version w/ EAPI 6?

Reply:
done